CharlieBackend.Api/Controllers/AccountsController.cs

```diff
@@ -126,31 +112,18 @@ public async Task<ActionResult> SignIn([FromBody] AuthenticationDto authenticati
                 {
                     return BadRequest();
                 }
-
-                var encodedJwt = _jWTGenerator.GenerateEncodedJwt(foundAccount, UserRole.Secretary);
-
-                authorization = "Bearer " + encodedJwt;
-
-                userRoleList.Add(UserRole.Secretary.ToString(), authorization);
-            }
-
-            if (foundAccount.Role == UserRole.Admin)
-            {            
-                var encodedJwt = _jWTGenerator.GenerateEncodedJwt(foundAccount, UserRole.Admin);
-                authorization = "Bearer " + encodedJwt;
-
-                userRoleList.Add(UserRole.Admin.ToString(), authorization);
             }
-            
             var response = new
             {
                 first_name = foundAccount.FirstName,
                 last_name = foundAccount.LastName,
-                role = foundAccount.Role,
-                role_list = userRoleList
+                role = foundAccount.Role.ToString()
             };
-
-            Response.Headers.Add("Authorization", authorization);
+            foreach (KeyValuePair<string, string> record in userRoleToJwtToken)
```

Comment:
If I remember, it was decided to put tokens into the response body, but not in headers.

Reply:
Ok, I've changed it towards more like it was, but I think this matter requires further discussion. We need to put all the tokens we have in body of response, that's clear. But do we have to leave at least one token in header? If so I've already done it. If we don't - I can remove this feature.